### PR TITLE
Orbit menu now shows assigned roles to antaghud users

### PR DESCRIPTION
**Pull request by Contrabang**
## What Does This PR Do

Orbit menu now shows assigned roles to antaghud users. This means you get a tooltip when you hover over someone to see what their assigned role is, with a little icon. It also lets you search by assigned role, so you can see how many of a specific role there is, such as security officers.

Right now, its currently locked behind antaghud, but it could easily be moved to all observers.

## Why It's Good For The Game
QOL for observers, so that you don't need to cross reference with the crew manifest.

## Images of changes

image
image
image
image
image
image

## Testing

See above

## Changelog
:cl:
add: Observers with antaghud can now view assigned roles in the Orbit menu
/:cl:

### CODE REVIEW

**Comment by Danchi299** [comment]
why only antagHUD users?

File: tgui/packages/tgui/interfaces/Orbit.js [hunk 1]
```diff
@@ -69,6 +76,25 @@ const OrbitedButton = (props, context) => {
...
+              as="img"
+              mr="0.5em"
+              // mt={0.5}
+              className={classes([
+                'orbit_job16x16',
+                thing.assigned_role_sprite,
+              ])}
+            />{' '}
+            {thing.assigned_role}
+          </Flex>
```
Comment (AyIong) [inline tgui/packages/tgui/interfaces/Orbit.js:92]:
```suggestion
          <Stack>
            <Box
              as="img"
              mr="0.5em"
              className={classes([
                'orbit_job16x16',
                thing.assigned_role_sprite,
              ])}
            />{' '}
            {thing.assigned_role}
          </Stack>
```
I don't see the point to comment that, and use Flex instead already imported Stack

File: tgui/packages/tgui/interfaces/Orbit.js [hunk 2]
```diff
@@ -4,17 +4,24 @@ import {
   Box,
   Button,
   Divider,
+  Flex,
```
Comment (AyIong) [inline tgui/packages/tgui/interfaces/Orbit.js:7]:
```suggestion
```
Use Stack

**Comment by lewcc** [comment]
oh hell yeah

**Comment by S34NW** [comment]
!merge_usptream

**Comment by S34NW** [comment]
!merge_upstream

File: code/modules/mob/dead/observer/orbit.dm [hunk 3]
```diff
@@ -15,6 +15,11 @@
 		ui = new(user, src, "Orbit", "Orbit")
 		ui.open()
 
+/datum/orbit_menu/ui_assets(mob/user)
+	return list(
+		get_asset_datum(/datum/asset/spritesheet/orbit_job)
+	)
```
[the remark this answers is not in the document]
Comment (Contrabang) [reply code/modules/mob/dead/observer/orbit.dm:21]: All ui_asset calls use this format
```
/obj/machinery/computer/cloning/ui_assets(mob/user)
	return list(
		get_asset_datum(/datum/asset/simple/cloning)
	)
```
```
/obj/machinery/prize_counter/ui_assets(mob/user)
	return list(
		get_asset_datum(/datum/asset/spritesheet/prize_counter)
	)
```

**Comment by S34NW** [comment]
!merge_upstream

**Review by S34NW — APPROVED** [review]

**Pull request merged**